sensor_msgs/PointCloud2 points
std_msgs/UInt32 top_k

---

Grasps grasps